lib/players/ai.rb

```diff
@@ -9,7 +9,8 @@ def initialize(board, checker, token, min_token)
   end
 
   def get_move(message, turns_remaining)
-    minimax(turns_remaining, -Float::INFINITY, Float::INFINITY, true)[1] + 1
+    depth = [turns_remaining, 9].min
```

Comment:
What's the reason for this depth min

Reply:
I was trying to speed up the minimax on the larger 4x4 board by having it use a shallower depth until more turns have been taken.

Reply:
Makes sense.  Removed it from this branch.